### DISCUSSION TITLE
Display always showing "Public-Pool.io" even if using solo.ckpool.org on ESP32-2432S082R. Why?

### DISCUSSION THREAD

**Discussion by Karkadam** [BitMaker-hub/NerdMiner_v2#360] [Ideas]
I have a bunch of ESP32-2432S082R all running the latest Nerdminer Software flashed on https://bitmaker-hub.github.io/diyflasher

Some of the Nerdminers i have configued to Public-Pool.io but the others i configured to solo.ckpool.org.

But all Nerdminers shows in display Public-Pool.io. Why? 

Why do the Nerdminer do not show the correct Pool Adress they are currently using?

I can not identify which miner is using Public-Pool.io and which is using solo.ckool.org

**Comment by kittydelta** [comment]
I am stuck with this problem as well. Waiting for an answer from someone who knows

**Comment by BitMaker-hub** [comment]
Hi guys, can you try to reflash the device with complete erase?
I think there is a problem on last release with some data remaining unchanged after a new configuration.

I'll have to look at this

**Reply by IAmSixNine** [reply]
I have tried that a few times, using https://espressif.github.io/esptool-js/  Connecting my device, deleting flash (clicked it 3 times), then installing the files. Still shows Public-pool.io on the dispaly. I just assumed it was a static image and no matter what pool you connected to it would just alway display the same thing.  One of the nice things about these is sometimes i have spare time and look to see what other pools will work with nerdminers. pool.vkbit.com works with them.

**Comment by nitroxgas** [comment]
Hi!
You mean at the bottom of the screen, right?
That's an image I created with the public pool in mind, so even the miner working at another pool will be displaying the image previously coded.
I'll try to improve this part of the code, ok?

**Reply by IAmSixNine** [reply]
Thank you for looking into this. 
So if we connect to a pool other than public-pool.io will it then (assuming changes are made in the code) show the proper miner information. I am talking about workers, best difficulty, total hash rate. 
I have a few on a BCH node now and all the stats are zeros for those three fields.

**Reply by IAmSixNine** [reply]
pool.vkbit.com no longer works with low hashrate devices. I mentioned it a few weeks ago but if someone reads this now it does not work. They had a site update and none of my nerdminers work with it anymore.
